plugins/Marketplace/resources/qml/Marketplace.qml

```diff
@@ -94,18 +94,20 @@ Window
                     id: pageSelectionTabBar
                     anchors.right: managePackagesButton.left
                     anchors.rightMargin: UM.Theme.getSize("default_margin").width
-
+                    height: UM.Theme.getSize("button_icon").height
                     spacing: 0
 
                     PackageTypeTab
                     {
                         width: implicitWidth
+                        padding: UM.Theme.getSize("default_margin").width/2
```

Comment:
We should probably move this padding into the PackageTypeTab element itself, right?

And why is it half the default margin, and not using the thinner margins from the theme?

Reply:
Sure, feel free to make those changes.